src/form/acp.jl

```diff
@@ -149,39 +144,39 @@ function constraint_mc_power_balance_slack(pm::AbstractUnbalancedACPModel, nw::I
     ungrounded_terminals = [(idx,t) for (idx,t) in enumerate(terminals) if !grounded[idx]]
 
     for (idx,t) in ungrounded_terminals
-        cp = JuMP.@NLconstraint(pm.model,
+        cp = JuMP.@constraint(pm.model,
               sum(  p[a][t] for (a, conns) in bus_arcs if t in conns)
             + sum(psw[a][t] for (a, conns) in bus_arcs_sw if t in conns)
             + sum( pt[a][t] for (a, conns) in bus_arcs_trans if t in conns)
-            ==
-              sum(pg[g][t] for (g, conns) in bus_gens if t in conns)
-            - sum(ps[s][t] for (s, conns) in bus_storage if t in conns)
-            - Pd[idx]
-            - ( # shunt
-                Gt[idx,idx] * vm[t]^2
-                +sum( Gt[idx,jdx] * vm[t]*vm[u] * cos(va[t]-va[u])
-                     +Bt[idx,jdx] * vm[t]*vm[u] * sin(va[t]-va[u])
+            + sum(-pg[g][t] for (g, conns) in bus_gens if t in conns)
+            + sum( ps[s][t] for (s, conns) in bus_storage if t in conns)
+            + Pd[idx]
+            + ( # shunt
+                (Gt[idx,idx] * vm[t]^2)
+                +sum( (Gt[idx,jdx] * vm[t]*vm[u] * cos(va[t]-va[u]))
+                     +(Bt[idx,jdx] * vm[t]*vm[u] * sin(va[t]-va[u]))
                      for (jdx,u) in ungrounded_terminals if idx != jdx)
             )
-            + p_slack[t]
+            - p_slack[t]
+            == 0.0
```

Comment:
Why were these changes necessary?

Reply:
We found a pattern that fixed some tests, and applied it across all the power (current) balance equations. It is possible that moving all items to one side of the equal sign is not strictly necessary, and only changing the sign in `-sum(pg)` to `+sum(-pg)` is what was needed to get the tests to pass.